crates/database/src/quotes.rs

```diff
@@ -91,7 +103,8 @@ WHERE
         (order_kind = 'buy' AND buy_amount = $5)
     ) AND
     order_kind = $6 AND
-    expiration_timestamp >= $7
+    expiration_timestamp >= $7 AND
+    onchain_signing_scheme IS NOT DISTINCT FROM $8
```

Comment:
Why `IS NOT DISTINCT FROM` instead of `!=`. I guess that's related to the column being nullable but not sure.

Reply:
Yes I am using `IS NOT DISTINCT FROM` as `=`, because of null. You can read about it here: 
https://docs.rs/sqlx/latest/sqlx/macro.query.html#nullability-bind-parameters